libs/functional/include/hpx/functional.hpp

```diff
@@ -6,4 +6,25 @@
 
 #pragma once
 
-#include <hpx/modules/functional.hpp>
+#include <hpx/functional/bind.hpp>
+#include <hpx/functional/bind_back.hpp>
+#include <hpx/functional/bind_front.hpp>
+#include <hpx/functional/function.hpp>
+#include <hpx/functional/invoke.hpp>
+#include <hpx/functional/invoke_fused.hpp>
+#include <hpx/functional/mem_fn.hpp>
+#include <hpx/functional/traits/is_bind_expression.hpp>
+#include <hpx/functional/traits/is_placeholder.hpp>
+
+namespace hpx {
+    using hpx::util::bind_back;
+    using hpx::util::bind_front;
+    using hpx::util::function;
```

Comment:
https://github.com/STEllAR-GROUP/hpx/pull/4707#issuecomment-639445396 and https://github.com/STEllAR-GROUP/hpx/pull/4707#issuecomment-639447936 for reference.

I really don't mind too much which way we go here. Most other utility classes have nonintrusive serialization so there's no need to choose. `any` is in a similar situation as `function`, with intrusive serialization. I'm happy to go with the majority opinion here.